pkg/isb/stores/jetstream/writer.go

```diff
@@ -230,7 +230,6 @@ func (jw *jetStreamWriter) asyncWrite(_ context.Context, messages []isb.Message,
 				} else {
 					writeOffsets[idx] = &writeOffset{seq: pubAck.Sequence, partitionIdx: jw.partitionIdx}
 					errs[idx] = nil
-					jw.log.Debugw("Succeeded to publish a message", zap.String("stream", pubAck.Stream), zap.Any("seq", pubAck.Sequence), zap.Bool("duplicate", pubAck.Duplicate), zap.String("domain", pubAck.Domain))
```

Comment:
Does this have performance impact when it's not in debug mode?

Reply:
it was showing up when I did cpu profile.